helper/helper.go

```diff
@@ -291,7 +304,7 @@ func DoCleanup() {
 			logVerbose("Encountered error closing error file: %v", err)
 		}
 	}
-	err := flushAndCloseRestoreWriter("Current writer pipe on cleanup", 0)
+	err := new(Helper).flushAndCloseRestoreWriter("Current writer pipe on cleanup", 0)
```

Comment:
I suggest simply calling the `flushAndCloseRestoreWriter` function, in which we will use the `Helper` instance that would have been created before.